core/server/worker/src/main/java/alluxio/worker/netty/DataServerBlockReadHandler.java

```diff
@@ -170,13 +170,15 @@ private void openBlock(Channel channel) throws Exception {
       return;
     }
 
-    int retryInterval = Constants.SECOND_MS;
+    int retryInterval = Constants.SECOND_MS * 2;
```

Comment:
I am not sure. I was trying this to see whether this can help increase the throughput.  I guess it probably does not matter much. I am reverting this for now. I will send another PR if I found that this should be modified.